### PR TITLE
Update UG to clarify email and notes validation

### LINKED ISSUE #319: [PE-D][Tester E] Parser for `addnote` not working properly
`addnote 1 n/first n/second ` should gives parser error for containing duplicate fields

-------------
Labels: `severity.Low` `type.FunctionalityBug`
original: rimgik/ped#1

### LINKED ISSUE #338: [PE-D][Tester A] Missing email ".com" test case
Currently, when adding or editing a contact, the email field does not enforce a ".com" constraint. While domain checks are implemented, additional validation to ensure that ".com" is included in the email would improve accuracy and consistency.

Perhaps introducing an additional validation check in the email parameter to ensure the presence of ".com" as part of the email address. This would align with the current domain checks and enhance input reliability.

The following image shows a Person with an email without ".com". 

-------------
Labels: `severity.Medium` `type.FeatureFlaw`
original: AmadeusEka/ped#2

### PR DESCRIPTION
**Pull request by ITLimJiaWei**
Closes #319 
Closes #338 

1.) Notes parsing 
2.) Email .com validation

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -98,6 +98,7 @@ A person can have any number of tags (including 0)
 - Note that Name, Phone Number, Email and Address are compulsory fields. 
 - We believe it is reasonable that a customer or business contacts will need to provide these fields as they are not 
 particularly sensitive as compared to Identification Number etc.
+- Email does not check for `.com` specifically at the end of each email as custom domain names exist. E.g. `@u.nus.edu`
```
Comment (sheenkerr) [inline docs/UserGuide.md:101]:
```suggestion
-The email validation does not check for the presence of a period (.) after the "@" symbol, and it does not verify any specific domain extension. It only ensures that the domain name after the "@" is at least two characters long.
```

Comment (sheenkerr) [reply docs/UserGuide.md:101]: Also would it be better to move to another section like I have done with the phone numbers? (where i created a new section called future features)

**Review by sheenkerr — CHANGES_REQUESTED** [review]

**Review by sheenkerr — APPROVED** [review]
LGTM

**Review by Nicholas-Cheng-De-Fei — APPROVED** [review]
LGTM

**Pull request merged**